prime-angular-frontend/src/app/modules/health-auth-site-reg/pages/site-management-page/site-management-page.component.ts

```diff
@@ -104,17 +79,8 @@ export class SiteManagementPageComponent implements OnInit {
   }
 
   public ngOnInit(): void {
-    this.getHealthAuthorities();
-  }
-
-  private getHealthAuthorities(): void {
-    const healthAuthorityId = this.authorizedUserService.authorizedUser.healthAuthorityCode;
-    this.busy = this.healthAuthorityResource.getHealthAuthorityById(healthAuthorityId)
-      .pipe(
-        map((healthAuthority: HealthAuthority) => this.healthAuthority = healthAuthority),
-        exhaustMap((healthAuthority: HealthAuthority) => this.healthAuthoritySiteResource.getHealthAuthoritySites(healthAuthority.id))
-      )
-      .subscribe((healthAuthoritySites: HealthAuthoritySite[]) => this.healthAuthoritySites = healthAuthoritySites);
+    this.healthAuthorityId = this.authorizedUserService.authorizedUser.healthAuthorityCode;
+    this.healthAuthoritySites$ = this.authorizedUserResource.getAuthorizedUserSites(this.healthAuthorityId);
```

Comment:
This should be called with the authorized user ID right? Not the health authority ID?